### PR TITLE
fix adapter string config

### PR DESCRIPTION
**Pull request by jeacott1**
allows push adapter to be properly configured via string config

### CODE REVIEW

**Comment by flovilmart** [comment]
Could you add a test that exemplify that behavior?

File: src/Adapters/AdapterLoader.js [hunk 1]
```diff
@@ -23,6 +23,9 @@ export function loadAdapter(adapter, defaultAdapter, options) {
     if (adapter.default) {
       adapter = adapter.default;
     }
+	if(options.adapter) {
```
Comment (flovilmart) [inline src/Adapters/AdapterLoader.js]: Alignment is off

Comment (jeacott1) [inline src/Adapters/AdapterLoader.js]: yeah - its a pain to test locally (windows) - so for tiny changes its easiest to just PR them and let travis let me know if its ok.
just trying a variation, and if its ok I'll add a test.

atm you cant configure a push adapter via string, which is a pain.

**Comment by flovilmart** [comment]
There is the test:win command that is specifically designed for windows.

**Comment by flovilmart** [comment]
Also, if you're using the CLi, you'll note that the parser is an objectParser not a moduleParser (unlike emailAdapter options). I suggest you open an issue first, with the proper error you encounter.

**Pull request closed without merging**